tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -1,25 +1,178 @@
+const assert = require('assert')
+const util = require('util')
+
 module.exports = {
   commands: {
+    clickLinkEditBtn: async function (linkName) {
+      const linkRowEditButtonSelector = util.format(this.elements.publicLinkEditButton.selector, linkName)
+      const linkRowEditButton = {
+        locateStrategy: 'xpath',
+        selector: linkRowEditButtonSelector
+      }
+      await this
+        .waitForElementVisible(linkRowEditButton)
+        .click(linkRowEditButton)
+    },
+    setExpiryDateXpath: function (year, month, day) {
```

Comment:
This functions doesn't set any date. Looks like it just returns xpaths. Please use appropriate function name.